src/Commands/Remote/WPCommand.php

```diff
@@ -26,15 +26,18 @@ class WPCommand extends SSHBaseCommand
      * @param array $wp_command WP-CLI command
      * @param array $options Commandline options
      * @option progress Allow progress bar to be used (tty mode only)
+     * @option int $retry Number of retries on failure
      * @return string Command output
      *
      * @usage <site>.<env> -- <command> Runs the WP-CLI command <command> remotely on <site>'s <env> environment.
      * @usage <site>.<env> --progress -- <command> Runs a WP-CLI command with a progress bar
+     * @usage <site>.<env> --retry=3 -- <command> Runs a WP-CLI command with up to 3 retries on failure
      */
-    public function wpCommand($site_env, array $wp_command, array $options = ['progress' => false])
+    public function wpCommand($site_env, array $wp_command, array $options = ['progress' => false, 'retry' => 0])
     {
         $this->prepareEnvironment($site_env);
         $this->setProgressAllowed($options['progress']);
-        return $this->executeCommand($wp_command);
+        $retries = isset($options['retry']) ? (int)$options['retry'] : 0;
```

Comment:
I think that `?:` is available in php 7.4. If so, this could just be `(int)$options['retry'] ?: 0;`

Reply:
Terminus uses this operator elsewhere.

Reply:
Do we want `(int)$options['retry'] ?: 0;` or `$retries = (int)($options['retry'] ?? 0);`?

Reply:
The first looks clearer to me, but both should behave the same. You may use the later if you prefer it.